### DISCUSSION TITLE
count of displayed Hover Information on POINT Data vs on PolyGon in Pyviz/ hvplot

### DISCUSSION THREAD

**Discussion by sebgeb1964** [holoviz/hvplot#1571] [General]
https://hvplot.pyviz.org/user_guide/Geographic_Data.html

In the Geopandas section you can find two examples displaying Data, but only the second one, with 'Polygon' Geodata intead of 'Points', is presenting all of the extra Information from the geopandas file, while hovering over the Map.

I've tried to manipulate the Points to Polygons but hope there is a better solution.

geopandas hover Information only lat,lon - https://i.stack.imgur.com/Eflz4.png

geopandas hover Information complete Dataframe  - https://i.stack.imgur.com/hWrwA.png

**Comment by philippjfr** [comment]
You can declare additional columns to add to the hover with `hover_cols`. The inconsistency between the two plot types isn't ideal though.
